group/mod/int.go

```diff
@@ -302,6 +292,16 @@ func (i *Int) Pick(rand cipher.Stream) kyber.Scalar {
 	return i
 }
 
+// ByteOrder return the byte representation type (big or little endian)
+func (i *Int) ByteOrder() kyber.ByteOrder {
+	return i.BO
+}
+
+// GroupOrder returns the order of the underlying group
+func (i *Int) GroupOrder() *big.Int {
+	return big.NewInt(0).SetBytes(i.M.Bytes())
```

Comment:
Not sure if you mean: 
```suggestion
	return i.Clone().(*Int).M
```
but Clone() doesn't do a deep copy of `M`, this may be a problem if one were to accidently modify it, for instance the `Add` method of big.Int modify the underlying big number.

Reply:
Wow, Clone should probably be doing a deep copy, that might be a bug.
but then 
```
func (z *Int) Set(x *Int) *Int
```
is probably what to use instead of SetBytes, no?

Reply:
much more concise, sounds good to me, I can look into fixing Clone() as well

Reply:
Please do in another PR